cover.md

```diff
@@ -0,0 +1,59 @@
+# Code coverage for GoAWK
+
+GoAWK implements code coverage functionality, similar to the one built in the Golang itself (https://go.dev/blog/cover).
```

Comment:
Thanks for adding this doc! It's a good start. There a bunch of grammar and style edits I'd like to make here, but let's just get this PR merged as is, and I'll make the edits to `cover.md` separately afterwards.